src/views/List.jsx

```diff
@@ -1,6 +1,7 @@
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	// TODO - Use navigation similar to in Home.jsx to navigate to home page if no token or token is null
```

Comment:
Awesome job defining a `TODO`! Understanding that it's
1. a feature that's a natural extension of current functionality
1. it's not on the acceptance criteria
1. another team will probably have to deal with this lack of functionality

shows great developer intuition! 10/10 foresight + teamwork here👏